### ISSUE TITLE
It would be nice to have a command line wrapper exe

### ISSUE THREAD

**Issue by UnixJunkie** [chembl/ChEMBL_Structure_Pipeline#14]
For example:
```
std -i input.smi -o output_std.smi [-np NPROCS]
```
The option to process molecules in parallel on a given number of CPUs is also
for people working with large files.

**Comment by UnixJunkie** [comment]
there is a PR for this one!
